File: INICIANTE/JAVASCRIPT/1037.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var prompt = function(texto) { return lines.shift(); };

var valor = prompt("valor")
if( 25 >= valor && valor >= 0){
    console.log("Intervalo [0,25]");
}else if( 50 >= valor && valor > 25){
    console.log("Intervalo (25,50]");
}else if( 75 >= valor && valor > 50){
    console.log("Intervalo (50,75]");
}else if( 100 >= valor && valor > 75){
    console.log("Intervalo (75,100]");
}else if( valor > 100 || valor < 0){
    console.log("Fora de intervalo")
}

File: INICIANTE/JAVA/1037.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        double valor;
        Scanner sc = new Scanner(System.in);
        valor = sc.nextDouble();
        if( 25 >= valor && valor >= 0){
            System.out.println("Intervalo [0,25]");
        }else if( 50 >= valor && valor > 25){
            System.out.println("Intervalo (25,50]");
        }else if( 75 >= valor && valor > 50){
            System.out.println("Intervalo (50,75]");
        }else if( 100 >= valor && valor > 75){
            System.out.println("Intervalo (75,100]");
        }else if( valor > 100 || valor < 0){
            System.out.println("Fora de intervalo");
        }
        sc.close();
    }
}
